packages/eslint-config-react/index.js

```diff
@@ -5,11 +5,6 @@ module.exports = {
       jsx: true,
     },
   },
-  settings: {
-    react: {
-      version: 'detect',
-    },
-  },
```

Comment:
caused by updates in eslint-plugin-react; we would have run into this the next time we bumped. Related issue: https://github.com/yannickcr/eslint-plugin-react/issues/2329